src/lib/statsv2.svelte

```diff
@@ -0,0 +1,179 @@
+  let sections = [
+    {
+      number: '1',
+      subtitle: 'largest clinic for homeless cats',
+      picture: 'statsv2/statImage1.png'
+    },
+    {
+      number: '115k',
+      subtitle: 'total cats spayed or neutered',
+      picture: 'statsv2/statImage2.png'
+    },
+    {
+      number: '12k',
+      subtitle: 'TNR volunteers trained',
+      picture: 'statsv2/statImage3.png'
+    },
+    {
+      number: '75%',
+      subtitle: 'cats are feral',
+      picture: 'statsv2/statImage4.png'
+    },
+    {
+      number: '89%',
+      subtitle: 'profits go directly to FixNation programs',
+      picture: 'statsv2/statImage5.png'
+    },
+    {
+      number: '21%',
+      subtitle: 'of cats abandoned because their of their residency',
+      picture: 'statsv2/statImage6.png'
+    }
+  ];
+
+  let screenWidth: number;
+
+  const updateScreenWidth = () => {
+    screenWidth = window.innerWidth;
+  };
+
+  onMount(() => {
+    if (typeof window !== 'undefined') {
+      window.addEventListener('resize', updateScreenWidth);
+      updateScreenWidth();
+    }
+  });
+
+  onDestroy(() => {
+    if (typeof window !== 'undefined') {
+      window.removeEventListener('resize', updateScreenWidth);
+    }
+  });
+</script>
+
+<h1>
+  FixNation’s mission is to reduce the population of homeless cats by sterilizing as many as
+  possible while demonstrating the effectiveness of Trap-Neuter-Return and colony management for the
+  humane care of homeless cats.
+</h1>
+<div>
+  {#if screenWidth > 675}
+    <div class="slider">
+      {#each sections as section}
+        <section class="section">
+          <img src={section.picture} />
+          <span class="rectangle" />
+          <div class="text">
+            <div class="header">{section.number}</div>
+            <div class="subtitle">{section.subtitle}</div>
+          </div>
+        </section>
+      {/each}
+    </div>
+  {:else}{/if}
+</div>
+
+<style>
+  h1 {
+    font-family: 'Avenir', sans-serif !important;
+    font-style: normal;
+    font-weight: 500;
+    text-align: center;
+    font-size: 1rem;
+    margin: 1vw;
+    padding: 2rem;
+  }
+  * {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 1;
+  }
+
+  section {
+    padding: 1rem;
+    min-width: 23rem;
+    height: auto;
+    scroll-snap-align: start;
+    text-align: center;
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  img {
+    width: 20rem;
+    height: auto;
+    margin-left: auto;
+    margin-right: auto;
+  }
+  .header {
+    color: #ffffff;
+    font-weight: 700;
+    letter-spacing: 0.02rem;
+    font-size: 4rem;
+    font-family: 'Rubik';
+  }
+  .subtitle {
+    color: #ffffff;
+    font-size: 1.3rem;
+    font-family: 'Avenir', sans-serif !important;
+    font-style: normal;
+    font-weight: 40;
+  }
+  .rectangle {
+    position: absolute;
+    width: 14rem;
+    height: 14rem;
+    margin: 1rem;
+    background: #008e7e;
```

Comment:
We have constants defined in our CSS for the main color schemes across the project. It's better to use these constants if possible to avoid duplicating code, and if we need to change the color scheme later, it's easier if we just have it defined in one plac.e

```suggestion
    background: var(--color-primary);
```